### DISCUSSION TITLE
error while running uc with `use_subprocess=False` !

### DISCUSSION THREAD

**Discussion by gitvishal96** [ultrafunkamsterdam/undetected-chromedriver#1528] [Q&A]
when I run uc.chorme(headless = True, use_subprocess=False) i run into this error -
File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 134, in _check_not_importing_main
    raise RuntimeError('''
RuntimeError:
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable.
        
        how do i fix it!Please help me!

**Comment by Inkthirsty** [comment]
i got this too
